jenkins/ci.suse.de/cloud-socok8s.yaml

```diff
@@ -6,3 +6,4 @@
     repo-credentials: c2350527-476a-45df-b406-84f028614682
     jobs:
       - '{name}-integration'
+      - '{name}-ansible-lint'
```

Comment:
I would prefer to call it just `lint`. we maybe add more linting beside the ansible one.

Reply:
I wasn wondering if we would create more than one linter and if it was on the same job or on a different one.... this seems to hint to the same job, Im ok with that :D